spectral/ruleset.yml

```diff
@@ -219,52 +264,14 @@ rules:
     severity: error
     then:
       function: ensureCommonHttpMethods
-  path-parameters-must-be-camel-case:
```

Comment:
renaming these to component-parameters (See above comment) and condensing the meaningful-*-names to a single rule that uses custom rule ensureMeaningfulParameterNames